server.js

```diff
@@ -2,14 +2,14 @@
 const express = require('express');
 const exphbs = require('express-handlebars');
 const sequelize = require('sequelize');
-const db = require('db');
-const {supervisorRouter, employeeRouter, indexRouter, hrRouter} = require('./routes');
+const {employeeRouter, indexRouter} = require('./routes');
 
 // initialize the express server
 const PORT = process.env.PORT || 5000;
 const app = express();
 
-
+// Serve static content for the app from the "public" directory in the application directory.
+app.use(express.static('public'));
```

Comment:
This serves everything in your `public` folder as if it were available in the root of your application.  Check the handlebars templates and the `/assets/...` references